### DISCUSSION TITLE
好奇，这里也会有毛怪吗？

### DISCUSSION THREAD

**Discussion by kukudexiaolan** [share121/inter-knot#2662] [其他]
绳网里会有毛怪吗？<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by shan-dgha** [comment]
好奇<img src="https://bbs-static.miyoushe.com/static/2023/10/10/84713208a6b3f8e2faaacb6766cae650_8216740381572967063.png" alt="蒂亚拉_心虚" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/10/10/5bc71020b76055d271f9e744daaf880e_5037006894041683415.png" alt="拉尼_晕" width="75" >
